src/views/tutorial-view/index.tsx

```diff
@@ -1,41 +1,114 @@
 import Content from '@hashicorp/react-content'
+import { useRouter } from 'next/router'
 import { MDXRemote, MDXRemoteSerializeResult } from 'next-mdx-remote'
-import { TutorialFullCollectionCtx as ClientTutorial } from 'lib/learn-client/types'
-import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import {
+  Collection as ClientCollection,
+  TutorialFullCollectionCtx as ClientTutorial,
+} from 'lib/learn-client/types'
+import SidebarSidecarLayout, {
+  SidebarSidecarLayoutProps,
+} from 'layouts/sidebar-sidecar'
+import Heading from 'components/heading'
 import MDX_COMPONENTS from './utils/mdx-components'
+import { formatTutorialToMenuItem, splitProductFromFilename } from './utils'
+import {
+  TutorialSidebar as Sidebar,
+  FeaturedInCollections,
+  CollectionCardProps,
+  Badges,
+  getIsBeta,
+} from './components'
 
-export interface TutorialViewProps extends Omit<ClientTutorial, 'content'> {
+export interface TutorialViewProps
+  extends Pick<
+    ClientTutorial,
+    | 'name'
+    | 'slug'
+    | 'readTime'
+    | 'productsUsed'
+    | 'edition'
+    | 'handsOnLab'
+    | 'video'
+  > {
+  collectionCtx: CollectionContext
   content: MDXRemoteSerializeResult
+  layout: TutorialSidebarSidecarProps
 }
 
-// @TODO update this interface once we have a better idea of the page needs
+export type CollectionContext = {
+  isDefault: boolean
+  current: ClientCollection
+  featuredIn?: CollectionCardProps[]
+}
+
+export type TutorialSidebarSidecarProps = Required<
+  Pick<SidebarSidecarLayoutProps, 'children' | 'headings' | 'breadcrumbLinks'>
+>
+
+/**
+ *
+ * Outstanding @TODOs
+ * - add canonical url if this is the default collection
+ * - fix: the toc overview linking isn't properly aligning
+ * - wire up instruqt embed for interactive labs
+ * - skeleton out the next / prev component after API endpoint is updated - https://app.asana.com/0/1201903760348480/1201932088801131/f
+ */
+
 export default function TutorialView({
   name,
+  slug,
   content,
+  layout,
+  readTime,
+  productsUsed,
+  edition,
+  handsOnLab,
+  video,
+  collectionCtx,
 }: TutorialViewProps): React.ReactElement {
+  const { asPath } = useRouter()
   return (
     <SidebarSidecarLayout
+      breadcrumbLinks={layout.breadcrumbLinks}
       sidebarSlot={
-        <>
-          {new Array(20).fill(null).map((_) => (
-            <div key={_}>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur
-              eros diam, fringilla ac malesuada vel, faucibus quis mauris.
-            </div>
-          ))}
-        </>
-      }
-      sidecarSlot={
-        <div>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur
-          eros diam, fringilla ac malesuada vel, faucibus quis mauris.
-        </div>
+        <Sidebar
+          title={collectionCtx.current.shortName}
```

Comment:
Just an FYI in case you didn't know about it, we have a `CurrentProduct` context that can be consumed anywhere in the app (as long as the `ProductSwitcher` is correctly hooked up) using `useCurrentProduct`. You could you use that here for example to get the current product's name! 😊

Reply:
Oh I see! I misread that part of the sidebar. 😆